File: app/OrderCancelled.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class OrderCancelled(AbstractEvent):
    id : int
    orderId : int
    status : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.orderId = None
        self.status = None

File: customer/src/main/java/shoesdelivery/domain/OrderCancelled.java
package shoesdelivery.domain;

import java.util.*;
import lombok.*;
import shoesdelivery.domain.*;
import shoesdelivery.infra.AbstractEvent;

@Data
@ToString
public class OrderCancelled extends AbstractEvent {

    private Long id;
    private Long orderId;
    private String status;
}
